integration/test_tdengine.py

```diff
@@ -0,0 +1,116 @@
+import os
+from datetime import datetime
+
+import pytest
+import taosrest
+from taosrest import ConnectError
+from taosws import QueryError
+
+from storey import SyncEmitSource, build_flow
+from storey.targets import TDEngineTarget
+
+url = os.getenv("TDENGINE_URL")
+user = os.getenv("TDENGINE_USER")
+password = os.getenv("TDENGINE_PASSWORD")
+has_tdengine_credentials = all([url, user, password]) or url.startswith("taosws")
```

Comment:
it should fix the test failure as well 
```suggestion
has_tdengine_credentials = all([url, user, password]) or (url and url.startswith("taosws"))
```